### PR TITLE
Update the unavailable sku style

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Update the unavailable sku style.

#### What problem is this solving?
The unavailable sku was with the same style than the available ones.

#### How should this be manually tested?
https://skusorted--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [X] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/SKUSelector/components/SelectorItem.js [hunk 1]
```diff
@@ -16,8 +16,14 @@ class SelectorItem extends PureComponent {
 
   render() {
     return (
-      <div className={`${VTEXClasses.SELECTOR__ITEM} di ba bw1 pointer flex items-center ${this.props.isSelected ? 'b--blue' : 'b--transparent'}`} onClick={this.handleClick}>
```
Comment (andreldsa) [inline react/components/SKUSelector/components/SelectorItem.js]: As I said, you reverted the previous changes.

`flex items-center ` and `b--blue`

File: react/components/SKUSelector/components/SelectorItem.js [hunk 2]
```diff
@@ -30,11 +36,15 @@ SelectorItem.propTypes = {
   children: PropTypes.node,
   /** Function that is called when the item is clicked */
   onClick: PropTypes.func,
+  /** Flag that indicates if the sku in available */
+  isAvailable: PropTypes.bool,
```
Comment (andreldsa) [inline react/components/SKUSelector/components/SelectorItem.js]: add to the readme

File: react/components/SKUSelector/components/SelectorItem.js (hunk 1 above)
Comment (vitoriaHeliane) [reply react/components/SKUSelector/components/SelectorItem.js]: Done!

File: react/components/SKUSelector/index.js [hunk 3]
```diff
@@ -35,16 +35,16 @@ class SKUSelector extends Component {
...
-        <SelectorManager 
-          title={this.props.title} 
+        <SelectorManager
+          title={this.props.title}
           onItemClick={this.handleSKUSelected}>
           {
             skuItems.map(skuItem => (
               skuItem.images.length > FIRST_INDEX &&
-              <SelectorItem key={skuItem.images[FIRST_INDEX].imageUrl}>
+              <SelectorItem key={skuItem.images[FIRST_INDEX].imageUrl} isAvailable={skuItem.sellers[0].commertialOffer.AvailableQuantity !== 0}>
```
Comment (andreldsa) [inline react/components/SKUSelector/index.js]: `> 0`, if different can be negative

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 - Slightly changed the layout of `ProductName` and `SkuSelector` to accomplish proposed design.
+- Style of the unavailable sku to be different from the available ones.
```
Comment (salesfelipe) [inline CHANGELOG.md]: Style of the unavailable sku to be different from the others.

File: react/components/SKUSelector/components/SelectorItem.js [hunk 5]
```diff
@@ -30,11 +36,15 @@ SelectorItem.propTypes = {
   children: PropTypes.node,
   /** Function that is called when the item is clicked */
   onClick: PropTypes.func,
+  /** Flag that indicates if the sku in available */
```
Comment (salesfelipe) [inline react/components/SKUSelector/components/SelectorItem.js]: /** Flag that indicates if the sku is available */

**Pull request merged**